contrib/babelfishpg_common/sql/varbinary.sql

```diff
@@ -143,7 +143,7 @@ AS 'babelfishpg_common', 'int4varbinary'
 LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
 
 CREATE CAST (INT4 AS sys.BBF_VARBINARY)
-WITH FUNCTION sys.int4varbinary (INT4, integer, boolean) AS ASSIGNMENT;
+WITH FUNCTION sys.int4varbinary (INT4, integer, boolean) AS IMPLICIT;
```

Comment:
These changes should also go to relevant upgrade file.

Reply:
That's the reason upgrade tests are failing.

Reply:
@shreyaxr3 Please add this changes in the upgrade script `contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--5.1.0--5.2.0.sql`